Framework/Source/API/StructuredBuffer.cpp

```diff
@@ -113,7 +117,7 @@ namespace Falcor
     void StructuredBuffer::getVariable(size_t offset, size_t elementIndex, VarType& value)
     {
         verify_element_index();
-        if(checkVariableByOffset<VarType>(offset, 1, mpReflector.get()))
+        if(checkVariableByOffset<VarType>(offset, 0, mpReflector.get()))
```

Comment:
What does the change from `1` to `0` here represent?

Reply:
0 means we don't accept an array. 1 means we accept an array with at least a single element.